_includes/masthead

```diff
@@ -3,7 +3,7 @@
 <div id="masthead-no-image-header">
 	<div class="row">
 		<div class="small-12 columns">
-			<a id="logo" href="{{ site.url }}" title="{{ site.title }} – {{ site.slogan }}">
+			<a id="logo" href="{{ site.url }}/" title="{{ site.title }} – {{ site.slogan }}">
```

Comment:
Sure, by all means just do that.

I seem to remember reading somewhere the 'jekyll-way' is to carry around
the {{site.url}} and {{site.baseurl}} command and use absolute referencing
(I do not remember why) but perhaps that's not needed. :)

On Thu, Oct 8, 2015 at 12:02 PM Daniele Procida <notifications@github.com>
wrote:

> In _includes/masthead
> <https://github.com/python-namibia/python-namibia.github.io/pull/26#discussion_r41500426>
> :
>
> > @@ -3,7 +3,7 @@
> >  <div id="masthead-no-image-header">
> >  	<div class="row">
> >  		<div class="small-12 columns">
> > -			<a id="logo" href="{{ site.url }}" title="{{ site.title }} – {{ site.slogan }}">
> > +			<a id="logo" href="{{ site.url }}/" title="{{ site.title }} – {{ site.slogan }}">
>
> Looks OK, though it would be more correct simply to use href="/" - that
> way we don't run into issues running on different sites.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/python-namibia/python-namibia.github.io/pull/26/files#r41500426>
> .
>